src/AppBundle/Service/Redirector.php

```diff
@@ -97,33 +97,50 @@ public function getFirstPageAfterLogin()
     }
 
     /**
-     * @return array [route, options]
+     * @param EntityDir\User $user
+     * @param string $currentRoute
+     * @return bool|string
      */
-    private function getLayDeputyHomepage(EntityDir\User $user, $enabledLastAccessedUrl = false)
+    public function getCorrectRouteIfDifferent(EntityDir\User $user, $currentRoute)
     {
+        // Redirect to appropriate homepage
+        if (in_array($currentRoute, ['lay_home','odr_index'])){
```

Comment:
odr_index should be merged into lay_home. For further discussion when everyone is back